src/main/java/uk/gov/hmcts/reform/civil/service/InitiateGeneralApplicationService.java

```diff
@@ -120,9 +122,7 @@ private GeneralApplication buildApplication(CaseData caseData, UserDetails userD
         } else {
             caseType = "UNSPEC_CLAIM";
         }
-        LocalDateTime deadline = deadlinesCalculator
```

Comment:
What is purpose of removing this?

Reply:
I didn't remove it. I have re located this code place at line 156. it throws exception that field is used after... more than 5 number of line.... something like that. not i remember well.